onnxruntime/core/providers/cuda/cuda_graph.h

```diff
@@ -29,6 +40,11 @@ struct CUDAGraph {
   bool has_graph_ = false;
   bool has_graph_exec_ = false;
 
+  cudaGraph_t additional_graph_ = NULL;
```

Comment:
So is the idea that we are using `additional_graph_` for the annotated graph captures and `graph_` for the session level graph capture ?

Reply:
As long as we document it clearly, there is no confusion (since that is common sense for developer that an option has default value).

In design, we have one or mutliple graph captures, and each one has unique ID, and user can use unique ID to replay the graph.